pallets/order-book/src/lib.rs

```diff
@@ -175,6 +175,18 @@ pub mod pallet {
 			Error = DispatchError,
 		>;
 
+		/// Type for a market price feeder
+		type FeederId: Parameter + Member + Ord + MaxEncodedLen;
+
+		/// Account used to feed with market prices
+		type MarketFeederId: Get<Self::FeederId>;
```

Comment:
Do we know the Account ID used to feed market values upfront? Or should it be the root/council who determines the account using an extrinsic?

Reply:
@mustermeiszer

Reply:
First approach: extrinsic. Second approach: oracle-collection